docker/docker-cluster/metrictank.ini

```diff
@@ -271,6 +271,8 @@ write-queue-size = 100000
 max-stale = 0
 #Interval at which the index should be checked for stale series.
 prune-interval = 3h
+# disable cassandra index updates (for read nodes)
```

Comment:
this comment is the inverse of what the option actually does.

suggestion:
`# synchronize index changes to cassandra. not all your nodes need to do this. `